### PR TITLE
WCOR-1133: Create Group Manage media view

### PR DESCRIPTION
**Pull request by jmilden-git**
### ODE 120

**Use case**

As an editor or publisher, I want to manage the media with a group to which I belong.  Also need to be able to associate the group at the time I upload a media file.

**Implementation details**

Create a media entity view similar to the manage group content view. The manage group content view uses content revisions, but for media, you can use a media view instead of media revisions as we don't currently use workflow on media.

Use the group manage content view as a template for the type of fields you'll want to display in this table-based, admin-theme view.

Update media library upload form display, add "Displayed in" field. (also added other fields)

**Dependencies**

Need media that is associated to a group for testing.  Upload new or use existing.

**Test Plan**

- [x] Given media upload involvement on an ODE, import S3 secrets into ODE
- [x] Import a test image to media library, note the associated Group node number
- [x] When uploading an image / document / video to media the "Displayed in" field is now present as a select list.
- [x] Under Views, find the Manage Group Media view, go to edit more and use the preview feature
- [x] Using that node number from your test image for preview, inspect preview for file / fields
- [x] Fields should be: Thumbnail, Name, Media Type, Changed, Added by

### CODE REVIEW

File: config/default/core.entity_form_display.media.document.default.yml [hunk 1]
```diff
@@ -32,13 +32,9 @@ content:
...
   field_group:
     weight: 4
-    settings:
-      match_operator: CONTAINS
-      match_limit: 10
-      size: 60
-      placeholder: ''
+    settings: {  }
     third_party_settings: {  }
-    type: entity_reference_autocomplete
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.document.default.yml:41]: Group selection will require autocomplete widget--we will have far too many groups for a select list. Please revert.

File: config/default/core.entity_form_display.media.document.media_library.yml [hunk 2]
```diff
@@ -11,27 +11,54 @@ dependencies:
...
-  translation:
-    weight: 10
+  field_document_type:
+    type: options_select
+    weight: 2
+    region: content
     settings: {  }
     third_party_settings: {  }
+  field_group:
+    type: options_select
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.document.media_library.yml:28]: Group selection will require autocomplete widget--we will have far too many groups for a select list.

File: config/default/core.entity_form_display.media.image.default.yml [hunk 3]
```diff
@@ -28,13 +28,9 @@ content:
...
   field_group:
     weight: 4
-    settings:
-      match_operator: CONTAINS
-      match_limit: 10
-      size: 60
-      placeholder: ''
+    settings: {  }
     third_party_settings: {  }
-    type: entity_reference_autocomplete
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.image.default.yml:37]: Group selection will require autocomplete widget--we will have far too many groups for a select list. Please revert.

File: config/default/core.entity_form_display.media.image.media_library.yml [hunk 4]
```diff
@@ -20,6 +20,12 @@ targetEntityType: media
 bundle: image
 mode: media_library
 content:
+  field_group:
+    type: options_select
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.image.media_library.yml:24]: Group selection will require autocomplete widget--we will have far too many groups for a select list.

File: config/default/core.entity_form_display.media.video.default.yml [hunk 5]
```diff
@@ -37,13 +37,9 @@ content:
...
   field_group:
     weight: 6
-    settings:
-      match_operator: CONTAINS
-      match_limit: 10
-      size: 60
-      placeholder: ''
+    settings: {  }
     third_party_settings: {  }
-    type: entity_reference_autocomplete
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.video.default.yml:46]: Group selection will require autocomplete widget--we will have far too many groups for a select list. Please revert.

File: config/default/core.entity_form_display.media.video.media_library.yml [hunk 6]
```diff
@@ -17,6 +17,27 @@ targetEntityType: media
 bundle: video
 mode: media_library
 content:
+  field_group:
+    type: options_select
```
Comment (lgbridges) [inline config/default/core.entity_form_display.media.video.media_library.yml:21]: Group selection will require autocomplete widget--we will have far too many groups for a select list.

**Review by lgbridges — CHANGES_REQUESTED** [review]
Please implement autocomplete widgets for group selection on media items.

**Comment by lgbridges** [comment]
QA revealed needed changes. The media items mentioned in test plan weren't assigned to node 206, so perhaps those assignments happened in your local but as content they didn't happen in Dev. I went in to assign a group to the media items but noticed the field_group is using select list widget. These will need to be autocomplete widgets given the number of groups we'll be managing.

**Comment by jmilden-git** [comment]
@lgbridges I updated the test plan to hopefully make more sense.  Let me know if you run into any issues.

**Review by lgbridges — APPROVED** [review]

**Pull request merged**